buffer/src/main/java/io/netty/buffer/PoolArenaMetric.java

```diff
@@ -92,6 +92,13 @@ public interface PoolArenaMetric extends SizeClassesMetric {
      */
     long numHugeAllocations();
 
+    /**
+     * Return the number of chunks allocations done via the arena.
```

Comment:
Maybe we should return -1 by default and add to the javadocs that -1 is returned when "unknown" ? Same for all the other methods.